kraftwerk-core/src/test/java/fr/insee/kraftwerk/core/outputs/scripts/SASImportScriptTest.java

```diff
@@ -0,0 +1,120 @@
+package fr.insee.kraftwerk.core.outputs.scripts;
+
+import fr.insee.kraftwerk.core.metadata.Variable;
+import fr.insee.kraftwerk.core.metadata.VariableType;
+import fr.insee.kraftwerk.core.metadata.VariablesMap;
+import fr.insee.vtl.model.Dataset;
+import fr.insee.vtl.model.Structured;
+import org.junit.jupiter.api.Test;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class SASImportScriptTest {
+
+    @Test
+    void generateSASImportScript_simpleCase() {
+        // Given
+        String idName = "ID";
+        String fooName = "FOO";
+        //
+        String tableName = "FOO_TABLE";
+        String csvFileName = "test_sas_script.csv";
+        Structured.DataStructure dataStructure = new Structured.DataStructure(List.of(
+                new Structured.Component(idName, String.class, Dataset.Role.IDENTIFIER),
+                new Structured.Component(fooName, Double.class, Dataset.Role.MEASURE)
+        ));
+        Map<String, VariablesMap> metadataVariables = new HashMap<>();
+        VariablesMap variablesMap = new VariablesMap();
+        variablesMap.putVariable(new Variable(idName, variablesMap.getRootGroup(), VariableType.STRING));
+        variablesMap.putVariable(new Variable(fooName, variablesMap.getRootGroup(), VariableType.NUMBER));
+        metadataVariables.put("FOO", variablesMap);
+
```

Comment:
This is a lot of objects to be able to instantiate an import script class. Seems the code pattern could be improved further.